doc/content/end-user-documentation/settings.rst

```diff
@@ -170,6 +170,10 @@ Views
     :name: FRONTEND_VIEW
     :path: lona.default_settings.FRONTEND_VIEW
 
+.. setting::
```

Comment:
Could you please add a short sentence of documentation here?

Reply:
Done